pcweb/templates/docpage/docpage.py

```diff
@@ -153,7 +155,9 @@ def docpage_footer(path: str):
                             padding="0px 10px",
                             white_space="nowrap",
                         ),
-                        href=f"https://github.com/reflex-dev/reflex-web/tree/main{path}.md",
+                        href=f"https://github.com/reflex-dev/reflex-web/blob/main/docs/getting-started/installation.md",
```

Comment:
We shouldn't hardcode this path since it changes for every dockage. It should instead reference the path. I think all we need to do is replace the original with `path.strip('/')` to strip of the extra trailing slash